app/harvesters/Itms/accounting_documents_harvester.rb

```diff
@@ -0,0 +1,92 @@
+require 'harvester_utils/downloader'
+
+class Itms::AccountingDocumentsHarvester
```

Comment:
nesuhlasim s tymto tvrdenim. pri nejakej zmene, ktora aktualne moze nastat, je potrebnych 5min, aby si tuto zmenu aplikoval pre vsetky harvestors. to nieje samozrejme v poriadku, ale ked to porovnam s casom, ktory bude potrebny, ak sa ukaze, ze dedicnost je v tomto pripade nevyhovujuca, tak mi to dava zmysel. goal v tomto pripade je to urobit cim jednoduchsie a pripadne navrhy designu nechat na neskor.